src/index.js

```diff
@@ -1,21 +1,25 @@
 import './style.css';
 import LIST from './modules/List.js';
 import TASK from './modules/Task.js';
```

Comment:
- [ ] Take a look at the image below. Right now when a task is added its `completed` property is set to **0**, However, as stated in the project requirements, the default value for a task's `completed` property should be **false**. Kindly fix this.
- [ ] Again, take a look at the image. When a task is added its `index` property is set to: `length of array - 1`. Meaning, if a fifth task is added, its index is 4. However, according to the requirements, the index of added items should be set to the length of the list so that the first item `index` is 1, the second item `index` is 2, and so on. Kindly fix this.

  ![zah-comp](https://user-images.githubusercontent.com/36542986/149245837-ecbeb3d8-3143-41b8-be37-db59edf453a5.jpg)

Reply:
Thank you, I seemed to have missed these details. I will implement them now